### PR TITLE
fix: distinguish named timeline range keyframe selectors

### PR DESCRIPTION
**Pull request by Pixel998**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/.github/blob/master/CONTRIBUTING.md).

#### AI acknowledgment

- [x] I did _not_ use AI to generate this PR.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

#### What did you do?

```css
/* eslint css/no-duplicate-keyframe-selectors: "error" */

@keyframes in-and-out {
  entry 0% {}
  entry 100% {}
  exit 0% {}
  exit 100% {}
}
```

#### What did you expect to happen?

No duplicate-selector errors, because each range/percentage pair is distinct.

#### What actually happened?

The rule used only the selector’s first component (`entry` or `exit`), so it incorrectly reported `entry 100%` and `exit 100%` as duplicates.

#### What is the purpose of this pull request?

This PR fixes false duplicate-selector reports for `@keyframes` selectors that include a named timeline range.

#### What changes did you make? (Give an overview)

Updated `no-duplicate-keyframe-selectors` to use the complete keyframe selector when detecting duplicates.

### CODE REVIEW

File: tests/rules/no-duplicate-keyframe-selectors.test.js [hunk 1]
```diff
@@ -396,5 +402,21 @@ ruleTester.run("no-duplicate-keyframe-selectors", rule, {
 				},
 			],
 		},
+		{
```
Comment (DMartens) [inline tests/rules/no-duplicate-keyframe-selectors.test.js:405]: Can you please add more test cases for whitespace in between, e.g. "entry   0%" (otherwise we could just use `sourceCode.getText(selector)` instead of the stringify logic)

Comment (Pixel998) [reply tests/rules/no-duplicate-keyframe-selectors.test.js:405]: Added in https://github.com/eslint/css/pull/507/commits/fdc4687b2885aea42551a0fab52622b4728a523f.

**Review by DMartens — APPROVED** [review]
Changes LGTM, thanks.
